File: PSP/Examen1EVA/cliente/src/main/java/ui/common/BasePantallaController.java
package ui.common;

import ui.pantallas.pantallamain.PantallaMainController;

public class BasePantallaController {
    private PantallaMainController PantallaMainController;

    public ui.pantallas.pantallamain.PantallaMainController getPrincipalController() {
        return PantallaMainController;
    }

    public void setPantallaMainController(PantallaMainController PantallaMainController) {
        this.PantallaMainController = PantallaMainController;
    }

    public void principalCargado() {

    }
}

File: PSP/ClienteGraphQL/src/main/kotlin/ui/pantallas/common/BasePantallaController.kt
package ui.pantallas.common

import ui.pantallas.mainscreen.MainController

class BasePantallaController {
    private var pantallaMainController: MainController? = null

    fun getPrincipalController(): MainController? {
        return pantallaMainController
    }

    fun setPantallaMainController(pantallaMainController: MainController?) {
        this.pantallaMainController = pantallaMainController
    }

    fun principalCargado() {}
}
